### PR TITLE
Add support for env vars / envFrom to Helm chart

### PR DESCRIPTION
**Pull request by shortypowers**
#### What this PR does / why we need it:
This makes it possible set configuration variables individually (DATADOG_ACCESS_KEY, DATADOG_API_KEY, DATADOG_PRIVATE_KEY, DATADOG_SECRET_ACCESS_KEY) from a Secret or ConfigMap or to set other generic environment variables

#### Which issue this PR fixes
n/a

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Documentation has been updated with helm-docs (run: `.github/helm-docs.sh`)
- [x] Chart Version bumped
- [x] `CHANGELOG.md` has beed updated
- [x] Variables are documented in the `README.md`

### CODE REVIEW

File: charts/synthetics-private-location/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Datadog changelog
 
+### 0.4.0
+* Add 'envFrom' and 'env' to support configuration via environment variables
```
Comment (clamoriniere) [inline charts/synthetics-private-location/CHANGELOG.md:4]:
```suggestion

* Add 'envFrom' and 'env' to support configuration via environment variables
```

File: charts/synthetics-private-location/values.yaml [hunk 2]
```diff
@@ -67,3 +67,15 @@ configFile: "{}"
...
+envFrom: []
+#   - configMapRef:
+#       name: <CONFIGMAP_NAME>
+#   - secretRef:
+#       name: <SECRET_NAME>
+
+# env -- Set environment variables
+env: []
+#   - name: <ENV_VAR_NAME>
+#     value: <ENV_VAR_VALUE>
```
Comment (clamoriniere) [inline charts/synthetics-private-location/values.yaml:81]:
```suggestion
#     value: <ENV_VAR_VALUE>

```

**Review by clamoriniere — COMMENTED** [review]
2 small nits, otherwise looks good 👍

**Review by clamoriniere — APPROVED** [review]
Thanks for this contribution.

**Review by m-rousse — APPROVED** [review]
:+1: Thanks!

**Pull request merged**